configurator/configurator-impl/src/main/java/org/codice/ddf/admin/configurator/impl/ConfiguratorImpl.java

```diff
@@ -353,6 +357,23 @@ public String deleteManagedService(String configPid) {
         return context.getService(ref);
     }
 
+    @Override
```

Comment:
Built on top of this commit for some sources functionality. Will fix this on rebase when other PR is merged.